docs/recipes/react/example-forms.md

```diff
@@ -183,24 +181,25 @@ const SubmitButton = createComponent(sendFormFx.pending, (props, loading) => (
   </button>
 ))
 
-const onSubmit = sendFormFx.prepend(e => new FormData(e.target)) //transforming upcoming data, from DOM Event to FormData
+const App = () => {
+  const handleSubmit = e => {
+    e.preventDefault()
+    onSubmit()
+  }
 
-onSubmit.watch(e => {
-  e.preventDefault()
-})
-
-const App = () => (
-  <form onSubmit={onSubmit}>
-    Login: <input name="name" />
-    <br />
-    Password: <input name="password" type="password" />
-    <br />
-    <Loader />
-    <SubmitButton />
-  </form>
-)
+  return (
+    <form onSubmit={onSubmit}>
+      Login: <input name="name" />
+      <br />
+      Password: <input name="password" type="password" />
+      <br />
+      <Loader />
+      <SubmitButton />
+    </form>
+  )
+}
```

Comment:
Please pay attention for such changes, example should not become broken even if it was looking weird

```jsx
//transforming upcoming data, from DOM Event to FormData
const onSubmit = sendFormFx.prepend(e => new FormData(e.target)) 

onSubmit.watch(e => {
  e.preventDefault()
})

const App = () => {
  return (
    <form onSubmit={onSubmit}>
      Login: <input name="name" />
      <br />
      Password: <input name="password" type="password" />
      <br />
      <Loader />
      <SubmitButton />
    </form>
  )
}
```

If you don't like to call `preventDefault` from `watch`, then you might replace that with a better example